Lib/importlib/util.py

```diff
@@ -5,18 +5,25 @@
 from ._bootstrap import spec_from_loader
 from ._bootstrap import _find_spec
 from ._bootstrap_external import MAGIC_NUMBER
```

Comment:
This is a public API exposing `MAGIC_NUMBER` so it's probably best to ignore my previous comment about changing the name of the thing to avoid confusion over its type.